wrapper/src/main/java/software/amazon/jdbc/Driver.java

```diff
@@ -122,9 +125,25 @@ public Connection connect(final String url, final Properties info) throws SQLExc
     if (!StringUtils.isNullOrEmpty(databaseName)) {
       PropertyDefinition.DATABASE.set(info, databaseName);
     }
+
     ConnectionUrlParser.parsePropertiesFromUrl(url, info);
+    final Properties props = PropertyUtils.copyProperties(info);
```

Comment:
can we rename the "info" variables? it is very unclear what it contains without going all the way up.

could it be originalProps and copiedProps?